src/components/Image/Image.st.css

```diff
@@ -1,10 +1,61 @@
+:import {
+  -st-from: "../../common/formatters.st";
+  -st-named: color, fallback, applyUnit;
+}
+
+/*Defaults*/
+:vars {
+  defaultBorderWidth: 0px;
+  defaultBorderRadius: 0px;
+  defaultContainerOpacity: 100%;
+  defaultImageOpacity: 100%;
+}
+
+/* Overrides */
+:vars {
+  /*
+    A background color by a color from the palette for the container
+  */
+  ContainerBackgroundColor: --overridable;
+  /*
+    A border color by a color from the palette for the container
+  */
+  ContainerBorderColor: --overridable;
+  /*
+    A border width in pixels for the container
+    @default 0px
+  */
+  ContainerBorderWidth: --overridable;
+  /*
+    A border radius in pixels for the container
+    @default 0px
+  */
+  ContainerBorderRadius: --overridable;
+  /*
+    An opacity between 0 to 100 for the container
+    @default 100%
+  */
+  ContainerOpacity: --overridable;
```

Comment:
Should the ContainerOpacity add opacity for the image as well?
If yes maybe we should change the name?

Reply:
According to the spec we should have both variables (one for the container, another for image)

Reply:
I asked @michalvardi regarding the opacity, maybe we should use alpha color for the background of the container instead `opacity`